apps/client-server/src/app/post/services/post-manager/post-manager.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Body, Controller, Get, Post } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { SubmissionId } from '@postybirb/types';
+import { PostManagerService } from './post-manager.service';
+
+class CancelPostDto {
+  submissionId: SubmissionId;
+}
+
+@ApiTags('post-manager')
+@Controller('post-manager')
+export class PostManagerController {
+  constructor(readonly service: PostManagerService) {}
+
+  @Post('cancel')
```

Comment:
issue: This is not a safe way to cancel a post really. We can just use the post-queue.controller's dequeue which makes sure that all db records are also properly handled.